simulation/traffic_simulator/src/math/transform.cpp

```diff
@@ -59,5 +60,34 @@ const geometry_msgs::msg::Pose getRelativePose(
 
   return ret;
 }
+
+const geometry_msgs::msg::Point transformPoint(
+  const geometry_msgs::msg::Pose & pose, const geometry_msgs::msg::Point & point)
+{
+  auto mat = quaternion_operation::getRotationMatrix(pose.orientation);
+  Eigen::VectorXd v(3);
+  v(0) = point.x;
+  v(1) = point.y;
+  v(2) = point.z;
+  v = mat * v;
+  v(0) = v(0) + pose.position.x;
+  v(1) = v(1) + pose.position.y;
+  v(2) = v(2) + pose.position.z;
+  geometry_msgs::msg::Point transformed;
+  transformed.x = v(0);
+  transformed.y = v(1);
+  transformed.z = v(2);
+  return transformed;
+}
+
+std::vector<geometry_msgs::msg::Point> transformPoints(
+  geometry_msgs::msg::Pose pose, std::vector<geometry_msgs::msg::Point> points)
```

Comment:
```
std::vector<geometry_msgs::msg::Point> transformPoints(
  geometry_msgs::msg::Pose pose, std::vector<geometry_msgs::msg::Point> points);
```
↓
```
std::vector<geometry_msgs::msg::Point> transformPoints(
  const geometry_msgs::msg::Pose & pose, const std::vector<geometry_msgs::msg::Point> & points);
```

Reply:
Fixed in https://github.com/tier4/scenario_simulator_v2/pull/633/commits/8a484b8ff81788c5188b6d779e81b84a87eff5be